docs/operations/etcd/cluster-migration-node-to-pod.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Etcd cluster migration"
+---
+
+This procedure explains how to migrate data from an etcd cluster running
+outside a Kubernetes cluster towards an installation using the Ondat etcd
+Operator. As a result the Ondat cluster will use the etcd running as Pods in
+Kubernetes.
+
+1. Backup the TLS artifacts from the current etcd cluster (if current etcd uses mTLS)
+    The Secret with the TLS material is usually named `storageos-etcd-secret`
+    or `etcd-client-tls`
+
+    ```bash
+    $ kubectl get secret \
+        -n storageos \
+        -oyaml \
+        etcd-client-tls > etcd-storageos-tls-secret-backup.yaml
+    ```
+
+1. Deploy etcd cluster in Kubernetes
+
+    It is required to select a storageClass other than ondat/storageos to run
+    etcd in the cluster.
+
+    ```bash
+    # Add ondat charts repo.
+    $ helm repo add ondat https://ondat.github.io/charts
+    # Install the chart in a namespace.
+    $ kubectl create namespace etcd-operator
```

Comment:
no need to create a namespace with `kubectl` - this can be achieved with `--create-namespace` helm flag,

```
helm install storageos-etcd ondat/etcd-cluster-operator \
        --namespace etcd-operator \
        --create-namespace \
        --set ondat.secret=storageos-etcd-secret-incluster \
        --set cluster.storageclass=standard # Choose the one according to your cluste
```

Reply:
added